testservices/novaservice/service_test.go

```diff
@@ -252,29 +252,6 @@ func (s *NovaSuite) TestAddServerTwiceFails(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `conflictingRequest: A server with id "test" already exists`)
 }
 
-// A control point can be used to change the status of the added server.
-func (s *NovaSuite) TestAddServerControlPoint(c *gc.C) {
```

Comment:
Updated the test to change the server name via the hook.  Changing the server status will 
no longer work due to changes to the test service to mimic the behavior of server status
of building, active and error.